ts/features/bonus/cgn/components/merchants/CgnMerchantsDiscountItem.tsx

```diff
@@ -48,9 +50,15 @@ const CgnMerchantDiscountItem: React.FunctionComponent<Props> = ({
   return (
     <TouchableDefaultOpacity style={[styles.listItem]} onPress={present}>
       <View style={[IOStyles.row, styles.container]}>
-        <View style={IOStyles.flex}>
+        <View style={[IOStyles.flex, IOStyles.row]}>
```

Comment:
This looks to be just a style issue. 

It is happening because in your attempt you're not adding the proper style to the badge component on the small case.  😄